torch/csrc/jit/serialization/import_export_functions.h

```diff
@@ -3,12 +3,13 @@
 // Functions that are used in both import and export processes
 namespace torch {
 namespace jit {
-void moduleMethodsTuple(
-    const Module& module,
-    std::vector<c10::IValue>& elements);
 IValue expect_field(IValue tup, const std::string& expected_name, size_t entry);
 std::string operator_str(
     const std::string& name,
     const std::string& overloadname);
+
+IValue Tup(std::vector<IValue> ivalues);
```

Comment:
Related to my comments in parse_bytecode.h/.cpp code

Are these methods (expect_field, Tup, Table) used in separate code (import vs export) with different implementations?
I'm just trying to understand the reason for this header to exist, and for us to have the implementations split between import.cpp and parse_bytecode.cpp

Reply:
They are functions that used in both export (or server) and import (or runtime) codes. 
It's independent of split import.cpp and parse_bytecode.cpp, which is more a layered design. parse_bytecode.cpp (leaner) is below import.cpp (for mobile phones).